languages/jelly/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:buster
```

Comment:
Use `FROM python:3-buster` to ensure the Python interpreter is always up-to-date, instead of relying on the one in Debian's repos

Reply:
If this is done, do we still need to make sure `python3` is installed through `apt`?

Reply:
No; the correct versions of Python and pip will be available immediately